File: Foundation/week_02-week_04/JavaBasicsProject/src/week_02/DataStructures/ListIntroduction2.java
package week_02.DataStructures;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class ListIntroduction2 {
  public static void main(String[] args) {

    List<String> listA = new ArrayList<>(Arrays.asList("Apple", "Avocado", "Blueberries", "Durian", "Lychee"));
    List<String> listB = new ArrayList<>();

    for (int i = 0; i < listA.size(); i++) {
      listB.add(listA.get(i));
    }
    System.out.println(listA.contains("Durian"));
    listB.remove("Durian");
    listA.add(4, "Kiwifruit");
    System.out.println(listA);
    boolean aIsBigger = listA.size() > listB.size();
    int indexOfAvocadoInA = listA.indexOf("Avocado");
    int indexOfDurianInB = listB.indexOf("Durian");
    listB.addAll(Arrays.asList("PassionFruit", "Pomelo"));
    System.out.println(listA.get(2));

  }

}

File: Foundation/week_02-week_04/PythonBasicsProject/week_02/data_structures/list_introduction_2.py
# Create a list ('List A') which contains the following values
# Apple, Avocado, Blueberries, Durian, Lychee
list_a = ["Apple", "Avocado", "Blueberries", "Durian", "Lychee"]

# Create a new list ('List B') with the values of List A
list_b = list_a

# Print out whether List A contains Durian or not
print("Durian" in list_a)

# Remove Durian from List B
list_b.remove("Durian")

# Add Kiwifruit to List A after the 4th element
list_a.append("Kiwifruit")

# Compare the size of List A and List B
if len(list_a) > len(list_b):
    print("list_a is bigger")
else:
    print("list_b is bigger")

# Get the index of Avocado from List A
if "Avocado" in list_a:
    list_a.index("Avocado")

# Get the index of Durian from List B
if "Durian" in list_b:
    list_b.index("Durian")

# Add Passion Fruit and Pomelo to List B in a single statement
list_b += ["Passion Fruit", "Pomelo"]

# Print out the 3rd element from List A
print(list_a[2])
